valohai_cli/commands/yaml/pipeline.py

```diff
@@ -12,13 +12,14 @@
 
 
 @click.command()
+@click.option('--yaml', default=None, help='Path to the YAML file to write to.')
```

Comment:
We're not only writing to the file, we're also reading it...
```suggestion
@click.option('--yaml', default=None, help='Path to the YAML file to update.')
```